public/content/lessons/2017/derivative-formulas-geometrically/index.mdx

```diff
@@ -1,10 +1,381 @@
+
+</Question>
+
+### Derivative of $f(x) = \frac{1}{x}$
+
+As another example, think of the function $f(x) = \frac{1}{x}$. Now, on the one hand, you could blindly try applying the power rule, since $\frac{1}{x}$ is the same as writing $x^{-1}$. That would involve letting that $-1$ hop down, leaving behind one less than itself, $-2$. 
+But let’s have some fun and see if we can reasonable this geometrically, rather than just plugging it through a formula.
+
+The value $1/x$ is asking “what number multiplied by $x$ equals $1$”, so here’s how I’d visualize it: Imagine a little rectangular puddle of water in two dimensions with area $1$. Let’s say that it’s width is $x$, which means the other must be $1/x$, since the total area is $1$.
+
+<Figure
+  image="figures/10.51.svg" 
+/>
+
+For example, if you increase $x$ to $3$, the other side must be squished down to $\frac{1}{3}$. And if $x=2$, the other side is forced to be $\frac{1}{2}$. 
+
+<Figure
+  image="figures/10.58.svg" 
+/>
+
+This is a nice way to think about the graph of $1/x$, by the way. If you think of the width $x$ of this puddle in the $xy$-plane, the corresponding output $1/x$, the height of the graph above that point, is whatever height the puddle must have to maintain an area of $1$.
+
+<Figure
+  image="figures/11.00.svg"
+  video="figures/11.11-11.24-trimmed.mp4" 
+/>
+
+For the derivative, imagine nudging the input $x$ up by a value $dx$. How must the height of this rectangle change so that the area remains unchanged at $1$? That is, increasing the width by $dx$ adds some new area to the right here, so the puddle must decrease in height by some $d(1/x)$ so that the area lost off the top here cancels that out.
+
+<Figure
+  image="figures/11.41.svg" 
+/>
+
+You should think of that $d(1/x)$ as being some tiny *negative* value, since it’s *decreasing* the height of this rectangle. And you know what? I’m gonna leave the last few steps here for you to pause and ponder. And once you work out $d(1/x)/dx$, compare it to what happens if you apply the power rule purely symbolically to $x^{-1}$.
+
+$$
+\frac{d(1 / x)}{d x}= ? ? ?
+$$
+
+
+## Comprehensive Questions
+
+Here are a couple questions to test your knowledge of derivatives and the past chapters.
+
+### Derivative of $f(x) = \sqrt{x}$
+
+Modeling the derivatives geometrically can be applied to many different types of functions. For example, see if you can reason your way through the derivative of $f(x) = \sqrt{x}$ which also can be written as $x^{\frac{1}{2}}$. Now, one way to find the derivative of this function is to recognize that it fits the mold of the power rule. Blindly applying the rule will produce an answer, but what does it *mean*?
+
+To start, try modeling function using geometry.
```

Comment:
```suggestion
Approaching this question with geometry is, by far, overkill. 
Frankly, it's a bit of a mind warp. 
But it does offer a satisfying explanation of an otherwise mostly symbolic fact, and more than that it's one more opportunity to flex our muscles in reasoning about how small nudges to one value can affect another.
```